doc/examples/gerchberg_saxton.py

```diff
@@ -7,11 +7,11 @@
 
 import numpy as np
 
-from light_prop.algorithms import GerchbergSaxton
-from light_prop.calculations import gaussian, get_gaussian_distribution
-from light_prop.lightfield import LightField
-from light_prop.propagation.params import PropagationParams
-from light_prop.visualisation import GeneratePropagationPlot, PlotTypes
+from lightprop.algorithms import GerchbergSaxton
+from lightprop.calculations import gaussian, get_gaussian_distribution
+from lightprop.lightfield import LightField
+from lightprop.propagation.params import PropagationParams
+from lightprop.visualisation import Plotter, PlotTypes
```

Comment:
Sorry I needed to write it somewhere:
```python
 params = PropagationParams.get_example_propagation_data()
    # Choose proper propagation parameters
  params.beam_diameter = 4
  params.matrix_size = 256
  params.pixel_size = 0.9
```

Can we just create a dict and wrap it with params? it's really confusing at this point

Reply:
And in line 45 this object is reused, it should be another object there, maybe I can create a task for it?

Reply:
As i mentioned offline - I want to remove this class anyway xd